package.json

```diff
@@ -150,7 +150,7 @@
     "ng-packagr": "^19.0.0",
     "nitropack": "^2.10.0",
     "nx": "20.1.2",
-    "playwright": "^1.30.0",
+    "playwright": "^1.49.1",
```

Comment:
Needed to fix the [failing e2e tests](https://github.com/analogjs/analog/actions/runs/12873576990/job/35891655604?pr=1563) which broke due to github recently upgrading the ubuntu version to 24. Support for ubuntu 24 was [only added in playwright `1.45`](https://github.com/microsoft/playwright/issues/34342#issuecomment-2596812036)